docs/.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
```

Comment:
This feels like unnecessary duplication within one big repo, but I don't know the best way to go about extending these rules to our root config & applying it to this sub directory